### PR TITLE
Add member notes

### PR DESCRIPTION
**Pull request by MarcMorinville-Rewind**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,4 +8,5 @@
 - Support of '/lists/<id>/interest-categories'
 - Support of 'lists/<id>/interest-categories/<id>/interests'
 - Support of '/lists/<id>/members'
+- Support of '/lists/<id>/members<id>/notes'
```
Comment (julian-huff) [inline CHANGELOG.md]: Missed a slash

File: lib/mailchimp_api/resources/member.rb [hunk 2]
```diff
@@ -10,6 +10,16 @@ class Member < Base
 
     self.prefix = '/3.0/lists/:list_id/'
 
+    # The path to get notes is '/3.0/lists/:list_id/members/:member_id'
+    # Unfortunately, ActiveResource does not support nested resources, only single parent resources:
+    #
+    # https://github.com/rails/activeresource/blob/4accda8bc03ceae0ad626f8cec0b4751e89a58ad/lib/active_resource/associations.rb#L151
+    #
+    # Using a has_many, there is no way to include the `list_id`, so we just create our own notes method.
+    def notes
```
Comment (julian-huff) [inline lib/mailchimp_api/resources/member.rb]: Should this support pagination?

**Review by julian-huff — APPROVED** [review]

File: lib/mailchimp_api/resources/member.rb (hunk 2 above)
Comment (MarcMorinville-Rewind) [reply lib/mailchimp_api/resources/member.rb]: Hmm. Hadn't considered. Shouldn't all endpoints support pagination?

Comment (julian-huff) [reply lib/mailchimp_api/resources/member.rb]: I'm sure the endpoint does, but this method doesn't

**Review by smandegar — APPROVED** [review]

**Pull request merged**